README.md

```diff
@@ -163,6 +163,18 @@ You can open the match requirement by using the ```--node-count``` option to fin
 
 **_NOTE:_**  The ```cmr```, ```--hunter-analyze``` and ```--anomaly-detection``` flags are mutually exclusive. They cannot be used together because they represent different algorithms designed for distinct use cases.
 
+#### Ack known bugs
+To ack known regressions, you must provide a yaml file with the timestamp which the regression was identified at.
+
+```
+---
+ack :
+  - timestamp: 1733490603,
+    metric: "etcdCPU_avg"
```

Comment:
Should we add a reason/description file?
A relatively small string to link some jiras, describe the issue, etc.?